### DISCUSSION TITLE
Your uploaded proof has already been used previously.

### DISCUSSION THREAD

**Discussion by Ulfatnvz** [community/community#120227] [GitHub Education]
### Hi everyone,

I want to get student developer benefites and when I upload proof of my academic status this error can be happen: Your uploaded proof has already been used previously. I hadn't used this account for github or github education. How can I fix this problem?

**Comment by samus-aran** [comment]
Hi @Ulfatnvz 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
